functions/model/teams/tark/createLabelProperties.js

```diff
@@ -23,7 +23,7 @@
 
 const { setLabelProperties, getTeam, updateTeamDetails } = require("../lib");
 
-exports.createLableProperties = function(orgDomain, teamName, type, statusLabels, priorityLabels, difficultyLabels) {
+exports.createLabelProperties = function(orgDomain, teamName, type, statusLabels, priorityLabels, difficultyLabels) {
```

Comment:
Please check and verify the name changes in the places it's used

Reply:
createLabelProperties is used in createTeam api and createDefaultLabelProperties is used nowhere else.